docs/application/dotnet/guides/user-interface/nui/text.md

```diff
@@ -756,6 +756,105 @@ The following table lists the available `TextEditor` properties:
 | `TranslatableText`                 | String      | Specifies the `TranslatableText` property that sets the SID value. |
 
 
+### Use public APIs to copy, cut, paste, and select text in TextField or TextEditor
```

Comment:
Hello @shrouqsabah, I had a query regarding the heading.
The heading here seems to be a bit too long, we recommend using something concise for headings.

In this case, could you please clarify if the heading should emphasize the functionality of **Copy, cut, paste, and select text** or the usage of **public APIs**? Depending on your clarification I can provide an alternate for the heading or it would be better if you could change the heading to something concise.

Thank you.

Reply:
Hello @omar-srbd ,
We need to explain how to use APIs.
Here's the original request from ACR. 
**TCSACR-455** : ` Please add how to use new APIs. When you update the API guides, it will be good if you add how to use SelectedText API `.

Let's me know what you suggest :)
Thanks

Reply:
@omar-srbd  What do you think about this ( **How to use Clipboard APIs with Select Text** )?

Reply:
@shrouqsabah,

I have left another review incorporating your suggestion for the heading.
Please look into it and update accordingly.

Thanks.